README.md

```diff
@@ -151,10 +151,10 @@ Support for grouping your platform (android, ios) IDs into one place via annotat
 public By Item;
 ```
 
-Default locator is `By.id` but there is also support for `By.xpath`: 
+Default locator is `By.id` but there is also support for `By.xpath` and 'By.className':
 ```java 
 @AndroidFindBy(xpath = "//*[@text='Knock Knock']")
-@IOSFindBy(xpath = "//*[@text='Who's there?']")
+@IOSFindBy(className = "//*[@text='Who's there?']")
```

Comment:
Is this a working example for className? It looks like it would need to get updated to a class like `@IOSFindBy(className = "responseBox")`